charts/nfdiv-cms/Chart.yaml

```diff
@@ -1,5 +1,9 @@
-apiVersion: v1
+apiVersion: v2
 appVersion: "1.0"
-description: Divorce - Case Maintenance Service
+description: No Fault Divorce - Case Maintenance Service
 name: nfdiv-cms
-version: 1.2.9
+version: 1.3.0
```

Comment:
ideally should be 2.0.0 ? as the java chart version is major upgrade ?

Reply:
tbh, I'm not really sure what the point is of these versions but we can bump it big